plugins/hu.bme.mit.massif.simulink.cli/src/hu/bme/mit/massif/simulink/cli/CLIEMFCreator.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2013, Embraer S.A., Budapest University of Technology and Economics
```

Comment:
Please correct copyright: use 2010-2017 as date and your own employer as holder.